src/ape_accounts/__init__.py

```diff
@@ -1,6 +1,13 @@
 from importlib import import_module
 
-from ape.plugins import AccountPlugin, register
+from ape.plugins import AccountPlugin, Config, register
+
+
+@register(Config)
+def config_class():
```

Comment:
Wonder if this instead should be a "core config" item, e.g.
```yaml
display:
  transactions: abridged
```

Reply:
Mostly since behavior isn't just limited to `ape-accounts`

Reply:
Good point, I like that